### PR TITLE
Feature/convert

### PR DESCRIPTION
**Pull request by catalino-dev**
## Status
**READY**

## Description
Added:
- DEG (degrees) `Math.toDegrees(double angRad)`
- RAD (radians) `Math.toRadians(double angDeg)`
- ABS (absolute) `Math.abs(double a)`

## Todos
**TESTS**
- [x] DEG
- [x] RAD
- [x] ABS

## Steps to Test
```sh
javac CalculateAction.java
java Calculate deg/rad/abs <number_to_test>
```

### CODE REVIEW

File: CalculateAction.java [hunk 1]
```diff
@@ -41,11 +41,36 @@ public String calculate(String[] args) {
...
+		}
 	};
 
 	public abstract String calculate(String[] args);
 
+<<<<<<< HEAD:CalculateAction.java
+	public static CalculateAction of(String name) {
+=======
 	public static Optional<CalculateAction> of(String name) {
+>>>>>>> fe78f710368fee30fe6619b5912f609ff6b77a32:CalculateAction.java
```
Comment (sbanjoe25) [inline CalculateAction.java]: Please fix conflicts on this line here.

**Review by sbanjoe25 — CHANGES_REQUESTED** [review]

File: CalculateAction.java [hunk 2]
```diff
@@ -41,11 +41,36 @@ public String calculate(String[] args) {
...
+		@Override
+		public String calculate(String[] args) {
+			double abs = Math.abs(Double.valueOf(args[0]));
+			return String.valueOf(abs);
+		}
 	};
 
 	public abstract String calculate(String[] args);
 
+<<<<<<< HEAD:CalculateAction.java
```
Comment (stevenJhonGil) [inline CalculateAction.java]: I think you should remove these lines

**Comment by kana-ph** [comment]
@saviercoo 

Please add `deg2gon` and `rad2gon` conversions.

`1 gon` = `PI / 200 radians`
_or_
`1 gon` = `9/100 degrees`

Reference: https://en.wikipedia.org/wiki/Gradian

**Comment by catalino-dev** [comment]
@sbanjoe25: I already fixed the CalculateAction changes you requested. :)

**Review by stevenJhonGil — APPROVED** [review]
:+1:

**Review by sbanjoe25 — APPROVED** [review]

**Pull request merged**